activiti-core/activiti-engine/src/test/java/org/activiti/engine/test/api/task/TaskQueryTest.java

```diff
@@ -2260,6 +2260,24 @@ public void testNativeQuery() {
     assertThat(taskService.createNativeTaskQuery().sql("SELECT count(*) FROM " + managementService.getTableName(Task.class) + " T WHERE T.NAME_ = #{taskName}").parameter("taskName", "gonzoTask").count()).isEqualTo(1);
   }
 
+    @Deployment(resources = { "org/activiti/engine/test/api/task/TaskQueryTest.testProcessDefinition.bpmn20.xml" })
```

Comment:
Better name the file `TaskQueryTest. testTaskExecutionId.bpmn20.xml` in that case you won't need the `Deployment` annotation.